Student/FindStudentViewController.swift

```diff
@@ -1,43 +1,34 @@
-//
-//  findStudentViewController.swift
-//  Student
-//
-//  Created by Sahil Gupta on 18/07/22.
-//
-
 import UIKit
 
-class findStudentViewController: UIViewController,UITextFieldDelegate {
-
-    
+class FindStudentViewController: UIViewController,UITextFieldDelegate
+{
     @IBOutlet weak var rollID: UITextField!
-    
     @IBOutlet weak var findStudentSubmitButton: UIButton!
-    
     @IBOutlet weak var foundStudentName: UITextField!
-    
     @IBOutlet weak var foundStudentId: UITextField!
     @IBOutlet weak var foundStudentAddress: UITextField!
-    
     @IBOutlet weak var foundStudentEmailId: UITextField!
-    
     @IBOutlet weak var foundStudentPhoneNo: UITextField!
-    
     @IBOutlet weak var rollError: UILabel!
     
+    @IBOutlet weak var buttonToPage3: UIButton!
+    
+    
     @IBAction func rollNoEntered(_ sender: Any) {
         if let id = rollID.text
         {
             if let errorMessage = invalidId(id)
             {
                 rollError.text = errorMessage
                 rollError.isHidden = false
+                findStudentSubmitButton.isEnabled = false
             }
             else
             {
                 rollError.isHidden = true
             }
         }
+        checkForValidFSearch()
     }
     func invalidId (_ value : String)->String?
```

Comment:
what's the use of this function? Add spacing on both sides of '->' for readability